libnavigation-base/api/current.txt

```diff
@@ -0,0 +1,334 @@
+    method public java.util.List<com.mapbox.geojson.Point>? component3();
+    method public float component4();
+    method public float component5();
+    method public float component6();
+    method public double component7();
+    method public String? component8();
+    method public com.mapbox.navigation.base.trip.model.RouteStepProgress copy(int stepIndex, com.mapbox.api.directions.v5.models.LegStep? step, java.util.List<com.mapbox.geojson.Point>? stepPoints, float distanceRemaining, float distanceTraveled, float fractionTraveled, double durationRemaining, String? guidanceViewURL);
+    method public float getDistanceRemaining();
+    method public float getDistanceTraveled();
+    method public double getDurationRemaining();
+    method public float getFractionTraveled();
+    method public String? getGuidanceViewURL();
+    method public com.mapbox.api.directions.v5.models.LegStep? getStep();
+    method public int getStepIndex();
+    method public java.util.List<com.mapbox.geojson.Point>? getStepPoints();
+  }
+
+  public static final class RouteStepProgress.Builder {
+    ctor public RouteStepProgress.Builder();
+    method public com.mapbox.navigation.base.trip.model.RouteStepProgress build();
+    method public com.mapbox.navigation.base.trip.model.RouteStepProgress.Builder distanceRemaining(float distanceRemaining);
+    method public com.mapbox.navigation.base.trip.model.RouteStepProgress.Builder distanceTraveled(float distanceTraveled);
+    method public com.mapbox.navigation.base.trip.model.RouteStepProgress.Builder durationRemaining(double durationRemaining);
+    method public com.mapbox.navigation.base.trip.model.RouteStepProgress.Builder fractionTraveled(float fractionTraveled);
+    method public com.mapbox.navigation.base.trip.model.RouteStepProgress.Builder guidanceViewURL(String? guidanceURL);
+    method public com.mapbox.navigation.base.trip.model.RouteStepProgress.Builder step(com.mapbox.api.directions.v5.models.LegStep step);
+    method public com.mapbox.navigation.base.trip.model.RouteStepProgress.Builder stepIndex(int stepIndex);
+    method public com.mapbox.navigation.base.trip.model.RouteStepProgress.Builder stepPoints(java.util.List<com.mapbox.geojson.Point> stepPoints);
+  }
+
+}
+
+package com.mapbox.navigation.base.trip.notification {
+
+  public enum NotificationAction {
+    enum_constant public static final com.mapbox.navigation.base.trip.notification.NotificationAction END_NAVIGATION;
+  }
+
+  public interface TripNotification {
+    method public android.app.Notification getNotification();
+    method public int getNotificationId();
+    method public void onTripSessionStarted();
+    method public void onTripSessionStopped();
+    method public void updateNotification(com.mapbox.navigation.base.trip.model.RouteProgress routeProgress);
+  }
+
+}
+
+package com.mapbox.navigation.base.utils {
+
+  public final class NullUtilsKt {
```

Comment:
Could we avoid tracking this empty `public` class? 🤔 There are only top-level functions